content/articles/2018-07-talk-carpe-diem.markdown

```diff
@@ -0,0 +1,82 @@
+
+http://bit.ly
+
+## Title and closing slide
+
+Make a slide with the most important details on you and your thing. Include this very early in your talk (probably first?) and repeat some variant at the end. This slide should feature your shortlink and any relevant URLs or handles, e.g. Twitter, GitHub, website, or email address.
+
+DO THIS IN AN ABSOLUTELY HUGE FONT. Now is not the time for subtlety or mumbling "I know it's hard to read, but ...".
+
+Leave this slide up for a bigly amount of time, so that people have ample opportunity to get out their phones and fire up the camera. Do not flash this vital slide for 3 seconds, only to spend 2 minutes displaying a bulleted outline of your talk.
+
+## Feature slides
+
+Admit that people photograph and share individual slides, e.g., via Twitter. This can be a good thing, especially if you harness it for your goals! If an audience member is moved to share your thing, it is natural they want to enhance the basic message with an evocative slide or image. Think about how well your slides work with this well-established behaviour.
+
+If something would be terrible taken out of context or circulated widely, reflect on that and take appropriate measures.
+
+If you have a pithy and hilarious point, give that slide some extra attention.
+
+*Link to some guides on live-tweeting? If you'd be chuffed to see people tweet about your talk, view your talk through their eyes. This is like grant-writing: notice the words the funder uses to describe what they want to fund and repeat them back, verbatim. Find out what a live-tweeter needs and give them exactly that.*
+
+## Post it before the talk
+
+Choose your primary online home and secure a shortlink at it BEFORE YOUR TALK. Do not promise the slides or materials "after the talk". People are taking notes and photos right now, during your talk, and this is your best chance to tell them how to find your awesome thing when they get home.
+
+Even if you add more materials or update your slides after the talk, it is still worth it to publicize the shortlink during the talk. These tweets will live on for months and wrap-up blog posts will come out over the next couple of weeks. Most of the genuine visits to your thing will happen later, but you will never have such a comparable opportunity to put the URL in front of people's eyeballs.
+
+## Share slides on a slide-sharing platform
+
+Consider how easy it is for a wide audience to click through your slides with minimum friction. Slide-specific platforms, such as [SpeakerDeck](https://speakerdeck.com), are ideal for quick browsing and linking to specific slides.
+
+The way you interact with and present your slides is not necessarily a great distribution platform. Large PDFs in GitHub repo do not offer a pleasant browsing experience and raw HTML slides on GitHub are even worse, i.e. are not viewable.
```

Comment:
In short, think about how the people who you want to use your stuff, and make their lives easier!